components/Downloads/DownloadList.tsx

```diff
@@ -1,17 +1,19 @@
 import { FormattedMessage } from 'react-intl';
 import LocalizedLink from '../LocalizedLink';
 import { useNavigation } from '../../hooks/useNavigation';
-import type { NodeVersionData } from '../../types';
+import type { NodeReleaseData } from '../../types';
 import type { FC } from 'react';
 
-type DownloadListProps = Pick<NodeVersionData, 'node'>;
+type DownloadListProps = {
+  release: NodeReleaseData;
+};
 
-const DownloadList: FC<DownloadListProps> = ({ node }) => {
+const DownloadList: FC<DownloadListProps> = ({ release }) => {
```

Comment:
```suggestion
const DownloadList: FC<NodeReleaseData> = ({ versionWithPrefix, ... }) => {
```

As I mentioned before the props can be the release data itself 🤔

Reply:
Ah, can ignore these first...
I planned to address them 😄 
Usually I push the commits daily after I complete one sitting
My bad, I should make the PR draft first

Reply:
No worries :)

Reply:
Hi,

I want to clarify...

I understand your intention since there's no need to declare a new type if there's no need for it.
However, if we have these components: `SecondaryDownloadMatrix` composed of `DownloadList`.
```js
type SecondaryDownloadMatrixProps = {
  release: NodeReleaseData;
};
```
```js
type DownloadListProps = {
  release: NodeReleaseData;
};
```

We can remove `DownloadListProps` type and use `NodeReleaseData` as the type directly. However, it's not practical to do so in `SecondaryDownloadMatrix` because we can't do this in `SecondaryDownloadMatrixProps`.
```js
<DownloadList {...release} />
```
Basically we are Props Drilling the `release` data (in this case the components are receiving either `lts` or `current`). What do you think about this case?

They are many options on how to approach this but I am asking because I want to have the standards so everything is written consistently.

Reply:
Instead of prop drilling we can use the hook then and have a prop on these components that tells if we want lts or current

Reply:
Sorry for the back and forth.
Yeah, it's tidier in components design.
I have the habit of overthinking things when I dive into something new. 😓 

I'll go through the changes tomorrow and simplify things.

And thank you for the quick response :D

Reply:
Yeah, as I mentioned feel free to use the Hook, and pretty much the only prop you can have in the component is something like `{ releaseType: 'lts' | 'current' }`

Which then:

```tsx
const releases = useNodeReleases();

const currentRelease = releases[props.releaseType];
```

Something like that